config/locales/client.en.yml

```diff
@@ -1827,6 +1827,10 @@ en:
         title: "Topic timeline"
         content: "Scroll quickly through a post using the topic timeline."
 
+      post_menu:
+        title: "Post menu"
+        content: "See how you can like the post, share the post, bookmark it and many more!"
```

Comment:
The copy here is going to be very tricky because different sites may have different features enabled/disabled. But shares can be disabled (and also aren't useful on private communities). Bookmarks can often be disabled as well.